src/Proto3/Suite/DotProto/Generate.hs

```diff
@@ -156,9 +167,10 @@ renameProtoFile filename =
 --   messages and enums.
 renderHsModuleForDotProto
     :: MonadError CompileError m
-    => ([HsImportDecl],[HsDecl]) -> DotProto -> TypeContext -> m String
-renderHsModuleForDotProto extraInstanceFiles dotProto importCtxt = do
-    haskellModule <- hsModuleForDotProto extraInstanceFiles dotProto importCtxt
+    => StringType
+    -> ([HsImportDecl],[HsDecl]) -> DotProto -> TypeContext -> m String
+renderHsModuleForDotProto custom extraInstanceFiles dotProto importCtxt = do
+    haskellModule <- hsModuleForDotProto custom extraInstanceFiles dotProto importCtxt
```

Comment:
The type of argument added here is `StringType`, not `ImportCustomisation`, so perhaps the new argument should be named `stringType` instead of `custom`?

There are also a few other places where the same issue arises.

Reply:
fixed

Reply:
Thanks.